src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -282,6 +282,15 @@ const generateTaskRole = (taskRole, labels, config) => {
     gangAllocation = 'false';
     retryPolicy.fancyRetryPolicy = true;
   }
+  // calculate pod priority
+  // reference: https://github.com/microsoft/pai/issues/3704
+  let jobPriority = 127;
+  if (launcherConfig.enabledHived) {
+    jobPriority = 126 - parseInt(config.taskRoles[taskRole].hivedPodSpec.priority);
+  }
+  const jobCreationTime = Math.floor(new Date() / 1000) & (Math.pow(2, 23) - 1);
```

Comment:
>new Date() [](start = 37, length = 10)

Is new Date() returns msec instead of sec?

Reply:
yes

Reply:
yes